src/main/java/frc/robot/subsystems/shooter/FlywheelIOSim.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021-2024 FRC 6328
+// http://github.com/Mechanical-Advantage
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// version 3 as published by the Free Software Foundation or
+// available in the root directory of this project.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.FlywheelSim;
+
+public class FlywheelIOSim implements FlywheelIO {
+  private FlywheelSim sim = new FlywheelSim(DCMotor.getNEO(1), 1.5, 0.004);
+  private PIDController pid = new PIDController(0.0, 0.0, 0.0);
+
+  private boolean closedLoop = false; //closed loop is pid
+  private double ffVolts = 0.0;
+  private double appliedVolts = 0.0;
+
+  @Override
+  public void updateInputs(FlywheelIOInputs inputs) {
+    if (closedLoop) {
```

Comment:
personally I wouldn't do this `if (closedLoop)` thingy (especially since I doubt we'll use openLoop on the flywheel) and just setup the pid within a method like `setVelocity()`